### PR TITLE
Update typings for `prices` field

### PR DESCRIPTION
**Pull request by LWJerri**
Hello dear developers! I made some changes in the typings for the `prices` field, because the description is outdated and I added new info, e.g max objects in prices.

### CODE REVIEW

**Comment by LWJerri** [comment]
What do I need do to?

**Comment by KnorpelSenf** [comment]
Wait until I have time. I am a little busy these days, sorry. On first sight, these changes are excellent, but I want to go through the docs once more and actually check that nothing else was forgotten. As soon as I'm done with that, I'll approve and merge :)

File: src/core/api.ts [hunk 1]
```diff
@@ -2022,7 +2020,7 @@ export class Api<R extends RawApi = RawApi> {
      * @param payload Bot-defined invoice payload, 1-128 bytes. This will not be displayed to the user, use for your internal processes.
      * @param provider_token Payments provider token, obtained via BotFather
      * @param currency Three-letter ISO 4217 currency code, see more on currencies
-     * @param prices Price breakdown, a list of components (e.g. product price, tax, discount, delivery cost, delivery tax, bonus, etc.)
+     * @param prices Price breakdown, a list of components (e.g. product price and name), max 15 objects.
```
Comment (KnorpelSenf) [inline src/core/api.ts:2023]: I don't know where from you're taking this. https://core.telegram.org/bots/api#sendinvoice says that the current version is correct.

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]
There changes are incorrect.

**Comment by KnorpelSenf** [comment]
Closing due to inactivity.

**Pull request closed without merging**